packages/react-instantsearch-core/src/connectors/connectSmartSort.ts

```diff
@@ -0,0 +1,77 @@
+import createConnector, { ConnectedProps } from '../core/createConnector';
+import {
+  refineValue,
+  getCurrentRefinementValue,
+  getResults,
+} from '../core/indexUtils';
+
+function getId() {
+  return 'relevancyStrictness';
+}
+
+function getCurrentRefinement(
+  props: ConnectedProps<any>,
+  searchState,
+  context
+) {
+  const id = getId();
+  const currentRefinement = getCurrentRefinementValue(
+    props,
+    searchState,
+    context,
+    id
+  );
+
+  return currentRefinement;
+}
+
+export default createConnector({
+  displayName: 'AlgoliaSmartSort',
+
+  getProvidedProps(props, searchState, searchResults) {
+    const results = getResults(searchResults, {
+      ais: props.contextValue,
+      multiIndexContext: props.indexContextValue,
+    });
+
+    if (!results) {
+      return {
+        isVirtualReplica: false,
+        isSmartSorted: false,
+      };
+    }
+
+    return {
+      isVirtualReplica: !(results.appliedRelevancyStrictness === undefined),
+      isSmartSorted: !(
+        results.appliedRelevancyStrictness === undefined ||
+        results.appliedRelevancyStrictness <= 0 ||
+        results.appliedRelevancyStrictness > 100
+      ),
```

Comment:
```suggestion
      isSmartSorted: (
        results.appliedRelevancyStrictness !== undefined &&
        results.appliedRelevancyStrictness > 0
      ),
```

it can't be > 100 as far as I can tell